--- configure.ac.org	2014-11-05 12:11:56.000000000 +0100
+++ configure.ac	2014-11-06 09:50:43.000000000 +0100
@@ -40,6 +40,9 @@
 # Replace any occurrences of @APXS2@ with the value of $APXS2 in the Makefile.
 AC_SUBST(APXS2)
 
+# Use environment varilable APXS2_OPTS to pass params to APXS2 command
+AC_ARG_VAR(APXS2_OPTS, [Additional command line options to pass to apxs2.])
+
 # We need the curl library for HTTP callouts.
 PKG_CHECK_MODULES(CURL, libcurl)
 AC_SUBST(CURL_CFLAGS)
--- Makefile.in.org	2014-11-05 12:11:56.000000000 +0100
+++ Makefile.in	2014-11-06 09:53:36.000000000 +0100
@@ -53,7 +53,7 @@
 REDIS_LIBS=@HIREDIS_LIBS@
 
 src/mod_auth_openidc.la: $(SRC) $(HDRS)
-	@APXS2@ -Wc,"-DNAMEVER=\"@NAMEVER@\" @OPENSSL_CFLAGS@ @CURL_CFLAGS@ @JANSSON_CFLAGS@ @PCRE_CFLAGS@ $(REDIS_CFLAGS)" -Wl,"@OPENSSL_LIBS@ @CURL_LIBS@ @JANSSON_LIBS@ @PCRE_LIBS@ $(REDIS_LIBS)" -Wc,-Wall -Wc,-g -c $(SRC)
+	@APXS2@ @APXS2_OPTS@ -Wc,"-DNAMEVER=\"@NAMEVER@\" @OPENSSL_CFLAGS@ @CURL_CFLAGS@ @JANSSON_CFLAGS@ @PCRE_CFLAGS@ $(REDIS_CFLAGS)"  -Wl,"@OPENSSL_LIBS@ @CURL_LIBS@ @JANSSON_LIBS@ @PCRE_LIBS@ $(REDIS_LIBS)" -Wc,-Wall -Wc,-g -c $(SRC)
 
 
 # Building configure (for distribution)
@@ -64,7 +64,7 @@
 	tar -c --transform="s#^#@NAMEVER@/#" -vzf $@ $(DISTFILES)
 
 test/test: test/*.c $(JWT_SRC) $(JWT_HDRS)
-	`@APXS2@ -q CC` @OPENSSL_CFLAGS@ @APR_CFLAGS@ @JANSSON_CFLAGS@ -Isrc -o $@ test/*.c $(JWT_SRC) @APR_LIBS@ @OPENSSL_LIBS@ @JANSSON_LIBS@
+	`@APXS2@ @APXS2_OPTS@ -q CC` @OPENSSL_CFLAGS@ @APR_CFLAGS@ @JANSSON_CFLAGS@ -Isrc -o $@ test/*.c $(JWT_SRC) @APR_LIBS@ @OPENSSL_LIBS@ @JANSSON_LIBS@
 
 test-compile: test/test
 
@@ -73,7 +73,7 @@
 
 .PHONY:	install
 install: src/mod_auth_openidc.la
-	@APXS2@ -i -n mod_auth_openidc src/mod_auth_openidc.la
+	@APXS2@ @APXS2_OPTS@ -i -n mod_auth_openidc src/mod_auth_openidc.la
 
 .PHONY: distfile
 distfile:	@NAMEVER@.tar.gz
